### DISCUSSION TITLE
analog mode

### DISCUSSION THREAD

**Discussion by hajdutony** [gab-k/RP2040-Decoder#28] [Ideas]
Hi, I built several decoders and are working really good, I ordered 20 boards/parts from china

however I am interested in analog functionality also, running locomotives with this decoders on analog ...

do you think it is necessary to do some hardware changes, or can be fixed in sw only?

**Comment by gab-k** [comment]
You might need some Hardware changes for analog mode. The decoder has no way of knowing the direction/polarity due to the bridge rectifier. Also the microcontroller needs a minimum voltage.
